Content.Server/GameObjects/EntitySystems/Atmos/PlayerGasOverlay.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using Content.Shared.GameObjects.EntitySystems.Atmos;
+using Robust.Shared.Map;
+
+namespace Content.Server.GameObjects.EntitySystems.Atmos
+{
+    public sealed class PlayerGasOverlay
+    {
+        private readonly Dictionary<GridId, Dictionary<MapIndices, GasOverlayChunk>> _data = 
+            new Dictionary<GridId, Dictionary<MapIndices, GasOverlayChunk>>();
+        
+        private readonly Dictionary<GasOverlayChunk, TimeSpan> _lastSent = 
```

Comment:
Any reason you're not using `GameTick` values for all these time comparisons? They're much cheaper.

Reply:
No good reason, updated. I also think I used TimeSpans in AiReachableSystem which I'll remember to change when I go back to it.